routing/pathfind.go

```diff
@@ -399,7 +394,7 @@ func findPath(g *graphParams, r *RestrictParams, cfg *PathFindingConfig,
 	// charges no fee. Distance is set to 0, because this is the starting
 	// point of the graph traversal. We are searching backwards to get the
 	// fees first time right and correctly match channel bandwidth.
-	distance[target] = nodeWithDist{
+	distance[target] = &nodeWithDist{
```

Comment:
comment above needs to be slightly updated in the commit removing `ForEachNode` :)